src/ctia/task/migration/migrate_es_stores.clj

```diff
@@ -310,52 +319,5 @@
       (exit true)))
   (exit false))
 
-(def cli-options
```

Comment:
Can you comment on why the tools.cli shell was dropped?

...Not against it, simply it'd make the pr more complete

Reply:
I updated the PR description to make more explicit the fact that this PR moves migration arguments from the command line to the properties.
Is it better?

Reply:
> I updated the PR description to make more explicit the fact that this PR moves migration arguments from the command line to the properties.
Is it better?

Sure thing thanks!